src/mega_lehmer.rs

```diff
@@ -1,12 +1,16 @@
-use crate::felt::FeltTrait;
+use crate::{felt::FeltTrait, seed_generators::time_seed};
```

Comment:
add multiplier gen (it is in my branch i guess i will change what you did and then pull request it, then we can add it)